source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_mixed.jsonnet

```diff
@@ -63,6 +63,7 @@ local proxyTitle = {
 {
   type: 'Question',
   id: 'mixed-or-multiple-ethnic-group',
+  page_title: 'Mixed or Multiple ethnic group or background',
```

Comment:
Should `Multiple` be capitalised?

Reply:
They are as per google sheet. I already sent the quick launch to Donna and Laura to double check.